### DISCUSSION TITLE
Link plex login to admin

### DISCUSSION THREAD

**Discussion by mgrimace** [sbondCo/Watcharr#471] [Q&A] [answered]
Hi, 

I created an admin account, then toggled on plex in the settings. my plex account name is the same name as my watcharr admin name, and I'd like plex to be the SSO for my admin account. Is there a way to 'link' the accounts?

**Comment by IRHM** [comment, marked as answer]
Hi @mgrimace, thanks for starting this discussion!

Unfortunately, there currently is no way to link Plex (or Jellyfin) to an existing account.

You can still login with your Plex account (usernames are only unique for same account types) and give yourself admin on the new account (this guide will show you how to do that: https://watcharr.app/docs/server_config/restoring-admin). Your new Plex admin account will have all the same privileges as the first admin account, so you can ignore the first one you've made.

Hopefully you haven't already started using the first admin account and this is a suitable alternative.

Thanks! Please let me know if there's anything else I can help you with (or if you're stuck with the guide above, if you're going to follow it).

**Reply by mgrimace** [reply]
Thanks for the quick reply! I did exactly what you suggested, but since I had already done most of the work on my initial admin, I just exported from admin and imported to plex using the inbuilt export/inport - worked just fine. 

This will likely be a separate discussion - I'm curious if anyone has already tried to use Plex authentication via Authentik to pass a Plex login directly to watcharr? Basically, if Watcharr is reverse proxied and behind something like Authentik or Authelia, I would only have to authorize via Plex once ... I do this with Overseer and Calibre-Web etc., and it involves passing `X-authentik-username` header from the authorization service

**Reply by mgrimace** [reply]
Also, is it 'safe' to delete the watcharr admin user if the plex user is elevated to admin? And is there an easy way to delete them?

**Reply by IRHM** [reply]
No problem @mgrimace, glad the export/import worked for you!

I haven't used Authentik/Authelia, not 100% sure how they work, but you can feel free to make an issue for it and I or someone else may be able to take a look at some point. Might be related to #461?

Yep it would be safe to delete the first admin account, unfortunately it's not possible yet, I actually just made an issue for it now: [#473](https://github.com/sbondCo/Watcharr/issues/473). I use Jellyfin to login, I've got my first admin account hanging around as a 'backup' in my password manager that I'll probably never need.
